src/allomancy/java/leaf/cosmere/allomancy/common/manifestation/AllomancyBendalloy.java

```diff
@@ -27,21 +30,59 @@ public AllomancyBendalloy(Metals.MetalType metalType)
 	@Override
 	protected void applyEffectTick(ISpiritweb data)
 	{
-		int mode = getMode(data);
-
 		if (AllomancyEntityThread.serverShutdown)
 		{
 			//don't start up new threads if the server is shutting down
 			return;
 		}
 
+		int mode = getMode(data);
 		String uuid = data.getLiving().getStringUUID();
-		if (mode > 0 && !playerThreadMap.containsKey(uuid))
+
+		// data thread management
 		{
-			playerThreadMap.put(uuid, new BendalloyThread(data));
+			if (mode > 0 && !playerThreadMap.containsKey(uuid))
+			{
+				playerThreadMap.put(uuid, new BendalloyThread(data));
+			}
+
+			playerThreadMap.entrySet().removeIf(entry -> !entry.getValue().isRunning || AllomancyEntityThread.serverShutdown || entry.getValue() == null);
 		}
 
-		playerThreadMap.entrySet().removeIf(entry -> !entry.getValue().isRunning || AllomancyEntityThread.serverShutdown);
+		// data processing
+		{
+			// check if bendalloy is off or compounding
+			if (mode <= 0)
```

Comment:
same deal here, no need to check for off or compounding as user is confirmed normal burn mode at this point